tabbycat/participants/emoji.py

```diff
@@ -1570,6 +1570,126 @@ def populate_code_names_from_emoji(teams, overwrite=True):
+    ("🩺",	True , "Stethoscope"),
+    ("🪑",	True , "Chair"),
+    ("🪒",	True , "Razor"),
+    ("🟠",	False, "Orange Circle"),                    # dull
+    ("🟡",	False, "Yellow Circle"),                    # dull
+    ("🟢",	False, "Green Circle"),                     # dull
+    ("🟣",	False, "Purple Circle"),                    # dull
+    ("🟤",	False, "Brown Circle"),                     # dull
+    ("🟥",	False, "Red Square"),                       # dull
+    ("🟧",	False, "Orange Square"),                    # dull
+    ("🟨",	False, "Yellow Square"),                    # dull
+    ("🟩",	False, "Green Square"),                     # dull
+    ("🟦",	False, "Blue Square"),                      # dull
+    ("🟪",	False, "Purple Square"),                    # dull
+    ("🟫",	False, "Brown Square"),                     # dull
+    
+    # Unicode Version 13.0
+    ("🥲",	False, "Smiling Face with Tear"),           # too similar to another
+    ("🥸",	True , "Disguised Face"),
+    ("🤌",	False, "Pinched Fingers"),                  # potentially offensive
+    ("🫀",	True , "Anatomical Heart"),
+    ("🫁",	True , "Lungs"),
+    ("🥷",	True , "Ninja"),
+    ("🫂",	True , "People Hugging"),
+    ("🦬",	True , "Bison"),
+    ("🦣",	True , "Mammoth"),
+    ("🦫",	True , "Beaver"),
+    ("🦤",	True , "Dodo"),
+    ("🪶",	True , "Feather"),
+    ("🦭",	True , "Seal"),
+    ("🪲",	False, "Beetle"),                           # potentially offensive
+    ("🪳",	False, "Cockroach"),                        # potentially offensive
+    ("🪰",	False, "Fly"),                              # potentially offensive
+    ("🪱",	False, "Worm"),                             # potentially offensive
+    ("🪴",	True , "Potted Plant"),
+    ("🫐",	True , "Blueberries"),
+    ("🫒",	True , "Olive"),
+    ("🫑",	True , "Bell Pepper"),
+    ("🫓",	True , "Flatbread"),
+    ("🫔",	True , "Tamale"),
+    ("🫕",	False, "Fondue"),                           # too similar to another
+    ("🫖",	True , "Teapot"),
+    ("🧋",	True , "Bubble Tea"),
+    ("🪨",	True , "Rock"),
+    ("🪵",	True , "Wood"),
+    ("🛖",	False, "Hut"),                              # potentially offensive
+    ("🛻",	True , "Pickup Truck"),
+    ("🛼",	True , "Roller Skate"),
+    ("🪄",	True , "Magic Wand"),
+    ("🪅",	True , "Piñata"),
+    ("🪆",	True , "Nesting Dolls"),
+    ("🪡",	False, "Sewing Needle"),                    # dull
+    ("🪢",	True , "Knot"),
+    ("🩴",	True , "Thong Sandal"),
+    ("🪖",	False, "Military Helmet"),                  # potentially offensive
+    ("🪗",	True , "Accordion"),
+    ("🪘",	True , "Long Drum"),
+    ("🪙",	True , "Coin"),
+    ("🪃",	True , "Boomerang"),
+    ("🪚",	True , "Carpentry Saw"),
+    ("🪛",	True , "Screwdriver"),
+    ("🪝",	True , "Hook"),
+    ("🪜",	True , "Ladder"),
+    ("🛗",	False, "Elevator"),                         # dull
+    ("🪞",	False, "Mirror"),                           # dull
+    ("🪟",	False, "A"),                           # dull
```

Comment:
Careful! Don't think this was intentional! Duplicates for ignored emoji don't really matter anyways.